### PR TITLE
Remove FFI stuff

### PR DESCRIPTION
**Pull request by dalyd**

### CODE REVIEW

File: mongo.el [hunk 1]
```diff
@@ -256,77 +128,37 @@
...
-(defun mongo-login (url)
-  (interactive "sURL: ")
-  (if (= (length url) 0)
-      (setq mongo--url "mongodb://localhost:27017")
-    (setq mongo--url url))
-  (mongo-client-new mongo--url)
-  (mongo-get-database "test")
-  (mongo-get-collection "test" "test")
-  ; is kill-buffer-hook the rights place to do this?
-  (add-hook 'kill-buffer-hook 'mongo--quit))
```
Comment (s-alty) [inline mongo.el:328]: Nice, I like that the user doesn't have to call this anymore.

File: README.md [hunk 2]
```diff
@@ -19,57 +18,42 @@ __This code is completely unsupported. Use at your own risk.__
...
     pushd mongoc
-        curl -LO https://github.com/mongodb/mongo-c-driver/releases/download/1.8.1/mongo-c-driver-1.8.1.tar.gz
-        tar xzf mongo-c-driver-1.8.1.tar.gz
-        cd mongo-c-driver-1.8.1
-        ./configure --prefix="$MONGO_MODE_HOME/lib"
-        make
-        make install
-        cd ..
+        curl -LO https://github.com/mongodb/mongo-c-driver/releases/download/1.11.0/mongo-c-driver-1.11.0.tar.gz
+        tar xzf mongo-c-driver-1.11.0.tar.gz
```
Comment (s-alty) [inline README.md:34]: Is this specific version of the c driver required, or will it work with whatever the user has installed?

**Review by s-alty — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (dalyd) [reply README.md:34]: Probably not -- but I know this one works :)

Comment (dalyd) [reply README.md:34]: I'm sure there are semantic guarantees on version number for things not changing.

**Pull request merged**